charts/ocis/docs/values-desc-table.adoc

```diff
@@ -384,6 +384,24 @@ a| [subs=-attributes]
 a| [subs=-attributes]
 `"https://idp.owncloud.test/realms/ocis"`
 | Issuer URI of the OpenID Connect Identity Provider. If the IDP doesn't have valid / trusted SSL certificates, certificate validation can be disabled with the `insecure.oidcIdpInsecure` option.
+| features.externalUserManagement.oidc.roleAssignment
+a| [subs=-attributes]
++object+
+a| [subs=-attributes]
+`{"claim":"roles","enabled":false,"mapping":{"admin":"ocisAdmin","guest":"ocisGuest","spaceadmin":"ocisSpaceAdmin","user":"ocisUser"}}`
+| Configure OIDC role assignment. If activate oCIS will read the role assigment from the OIDC token, see  xref:{s-path}/proxy.adoc#automatic-role-assignments[Automatic Role Assignments]
```

Comment:
```suggestion
| Configure OIDC role assignment. If activated, oCIS will read the role assigment from the OIDC token, see  xref:{s-path}/proxy.adoc#automatic-role-assignments[Automatic Role Assignments]
```

Reply:
addressed in https://github.com/owncloud/ocis-charts/pull/181/commits/00c334d5fbf930629268fa9cb8564adb530cd0ea